### ISSUE TITLE
Колонка Published

### ISSUE THREAD

**Issue by RageXL** [sleeping-owl/admin#140]
Возможно ли сделать колонку с отображением статуса статьи как в V3

![1](https://cloud.githubusercontent.com/assets/6135368/8210730/039adc40-151c-11e5-9a23-0a398eac7c11.jpg)

**Comment by takemore** [comment]
Да, в документации есть хороший пример - http://sleeping-owl.github.io/ru/Columns/Custom_Columns.html
